microsoft-edge/dualengine/reference/accelerator.md

```diff
@@ -24,7 +24,8 @@ api_location:
 # struct ACCELERATOR
 
 > [!IMPORTANT]
-> The DualEngine API is a limited access feature. Contact dualengineapiaccess@microsoft.com for more information.
+> The DualEngine API is part of a Limited Access Feature (see [LimitedAccessFeatures class](https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.limitedaccessfeatures)). For more information or 

```

Comment:
The warnings on the PR are coming from the presence of the `en-us` language tag. This tag is not needed, and would prevent someone browsing learn.microsoft.com in a different language to access those links in their prefered language.

To fix this, you can just drop the entire beginning of the URL, all the way up to /uwp/:

`/uwp/api/windows.applicationmodel.limitedaccessfeatures`